libraries/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -106,7 +106,7 @@ export class RushConfigurationProject {
     const packageJsonFilename: string = path.join(this._projectFolder, FileConstants.PackageJson);
 
     try {
-      this._packageJson = JsonFile.load(packageJsonFilename);
+      this._packageJson = JsonFile.load(packageJsonFilename, true);
```

Comment:
Enable JSON validation during repo's components' `package.json` parsing.

Reply:
We normally would introduce an `options` parameter, since the meaning of `, true` is unclear to readers, and we may want to add additional optional parameters in the future.  I can fix this.

Reply:
@1000RR I pushed an update to your branch -- let me know what you think

Reply:
I think this works.  Please see my other note in a lower thread.

Reply:
We need your to sign the CLA in order to get this PR merged.